Modules/RosterManager/Options.lua

```diff
@@ -425,20 +431,9 @@ local function dynamic_item_values(roster, equationGet, equationSet, multiplierG
         name = CLM.L["Equation"]
     }
     order = order + 1
-    local wowpedia = "WoWpedia: |c43eeee00[Multiplier] * [item value]^2 * [slot multiplier]|r\n\n"
-    local epgpweb = "EPGPWeb: |c43eeee00[Multiplier] * 2^(ilvl/26 + (rarity - 4)) * [slot multiplier]|r\n\n"
+    local wowpedia = "WoWpedia: |c43eeee00[Multiplier] * [item value]^[Exponent] * [slot multiplier]|r\n\n"
```

Comment:
Since we are using `[Exponent]` and `[Base]` here in equation descriptions then both should also be used in the field description so users can simply understand they are related to it.